ern-cauldron-api/src/upgrade-scripts/0.0.0-1.0.0.js

```diff
@@ -0,0 +1,38 @@
+import CauldronApi from '../CauldronApi'
+
+export default async function upgrade (cauldronApi: CauldronApi) {
+  try {
+    log.info('Upgrading Cauldron schema from v0.0.0 to v1.0.0')
```

Comment:
these versions will change in future, better to parameterize them

Reply:
There is one upgrade script per new schema version, as the operations to upgrade to a new schema version will be different for each version. So this is the script to upgrade from schema version `0.0.0` to `1.0.0` of the schema `thus the script is named 0.0.0-1.0.0.js`. If we come up with a new schema `2.0.0` we will create a script `1.0.0-2.0.0.js`. Some of the things might be common for all the scripts, but we will refactor if needed, once we have more than one script ;)